Packs/Jira/Integrations/JiraEventsCollector/JiraEventsCollector.py

```diff
@@ -196,20 +182,17 @@ def main():
         get_events.run(max_fetch=1)
         demisto.results('ok')
 
-    elif command in ('fetch-events', 'JiraAuditRecords-get-events'):
+    elif command in ('fetch-events', 'jira-fetch-events'):
```

Comment:
Sorry about that, My bad.
```suggestion
    elif command in ('fetch-events', 'jira-get-events'):
```